doc/howtos/qemu.rst

```diff
@@ -0,0 +1,35 @@
+Using QEMU
+##########
+
+A Refkit wic image can be tested with QEMU as is. For that make sure that the
+`ovmf` target has been built::
+
+  $ bitbake ovmf
```

Comment:
FWIW, meta-intel added ovmf in ```EXTRA_IMAGEDEPENDS``` so it should be available. But I guess this won't hurt.